integrationtest/src/test/java/dk/kvalitetsit/hello/integrationtest/HelloIT.java

```diff
@@ -21,16 +25,102 @@ class HelloIT extends AbstractIntegrationTest {
         helloApi = new KithugsApi(apiClient);
     }
 
+    @BeforeAll
+    public static void setUpLocale() {
+        //On certain computers the language for Spring error messages defaults to Danish.
+        //This would cause certain tests where we compare error messages to strings to fail.
+        //This forces the language of error messages to English, preventing any issues.  
+        Locale.setDefault(Locale.ENGLISH);
+    }
+
     @Test
-    void testCallService() throws ApiException {
-        var input = "John Dow";
+    void testCallServiceWithName() throws ApiException {
+        //in V901__extra_data_for_integration_test.sql the name "Some Name" is set to be inserted into the db.
+        //here we test that we can get that name from the db.
+        var input = "Some Name";
+
+        var result = helloApi.v1HelloGet(input);
+
+        assertNotNull(result);
+        assertEquals(1, result.size());
+        boolean containsSomeName = result.stream()
+            .anyMatch(dbEntry -> "Some Name".equals(dbEntry.getName()));
+        assert(containsSomeName);
+    }
+
+    @Test
+    void testCallServiceWithNameNotInDB() throws ApiException {
+        //Test that calling the get method with a name that is not in the db returns an empty list
+        var input = "notindb";
+
+        var result = helloApi.v1HelloGet(input);
 
+        assertNotNull(result);
+        assertEquals(0, result.size());
+    }
+
+    @Test
+    void testCallServiceWithoutName() throws ApiException {
+        //in V901__extra_data_for_integration_test.sql the name "Some Name" is set to be inserted into the db.
+        //here we test that that is the only name we get from the db when we try to get everything from the db
+        String input = null;
+        
         var result = helloApi.v1HelloGet(input);
 
         assertNotNull(result);
-        assertEquals(input, result.getName());
-        assertNull(result.getiCanBeNull());
-        assertNotNull(result.getNow());
+        assertEquals(1, result.size());
+        boolean containsSomeName = result.stream()
+            .anyMatch(dbEntry -> "Some Name".equals(dbEntry.getName()));
+        assert(containsSomeName);
+    }
+
+    @Test
+    void testCallPostService() throws ApiException {
+        var request = new HelloRequest().name("John Dow");
+
+        var postResult = helloApi.v1HelloPost(request);
+        assertNotNull(postResult);
+        assertEquals(request.getName(), postResult.getName());
+        assertNull(postResult.getiCanBeNull());
+        assertNotNull(postResult.getNow());
+    }
+
+    @Test
+    void testCallPostAndGetService() throws ApiException {
```

Comment:
Er det ikke korrekt at vi i denne metode får testet både hent og læs med og uden name? Hvis det er tilfældet, får vi så ikke testet det hele to gange?